cohttp-eio/tests/test_client.ml

```diff
@@ -0,0 +1,124 @@
+module Net = Eio.Net
+module Stdenv = Eio.Stdenv
+module Switch = Eio.Switch
+open Cohttp_eio
+
+let conn env sw port resource_path =
```

Comment:
Taking `resource_path` here means that every time you want a different resource, you also need to open a new connection. Why not separate out the connection from individual requests?

Reply:
Done in 09c2d01be65454926d0a3260b199e88e92c7760c